docs/03-AcceptanceTests.md

```diff
@@ -310,13 +319,65 @@ $I->seeInCurrentUrl('user/1');
 $user_id = $I->grabFromCurrentUrl('~$/user/(\d+)/~');
 ```
 
-## Selenium WebDriver
+## WebDriver: Selenium Server or PhantomJS
 
 A nice feature of Codeception is that most scenarios are similar, no matter of how they are executed.
 `PhpBrowser` was emulating browser requests but how to execute such test in a real browser like Chrome or Firefox? 
 Selenium WebDriver can drive them so in our acceptance tests we can automate scenarios we used to test manually.
 In such tests we should concentrate more on **testing the UI** than on testing functionality.
 
+"[Selenium WebDriver](http://www.seleniumhq.org/projects/webdriver/)" is the name of an API (specified by the Selenium project)
+to drive browsers automatically. Codeception supports two implementations of this API: Selenium Standalone Server and PhantomJS
+
+### Selenium Standalone Server
+
+Selenium Server is a piece of software that "drives" (i.e. runs) your local browser by sending commands (i.e. your test cases) to it.
+
+Overview of components:
+```
+Codeception -> facebook/php-webdriver -> Selenium Server -> Gecko/Chrome Driver -> Browser (Firefox or Chrome)
+```
+
+Installation instructions:
+
+1. You need [Firefox](http://www.mozilla.com/) and/or [Chrome](https://www.google.com/chrome/browser/desktop/) :-)
+1. You need [Java](http://www.java.com/)
+1. Download [Selenium Standalone Server](http://www.seleniumhq.org/download/)
+1. Download [geckodriver](https://github.com/mozilla/geckodriver/releases) (for Firefox) and/or
+[ChromeDriver](https://sites.google.com/a/chromium.org/chromedriver/) and extract the executable to a folder where Selenium Server can find it
+(best guess: just drop it in the same folder where `selenium-server-standalone-xxx.jar` lives)
```

Comment:
Better to drop it into $PATH
Otherwise you need to specify path to the executable like this: `-Dwebdriver.chrome.driver=./chromedriver`
Anyway, this documentation was written in [WebDriver module reference](http://codeception.com/docs/modules/WebDriver#Local-Testing)

Maybe we should not duplicate it and have one source?